src/live.asp.net/TagHelpers/HtmlCommentTagHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace live.asp.net.TagHelpers
+{
+    [HtmlTargetElement("*", Attributes ="commented")]
+    public class HtmlCommentTagHelper : TagHelper
```

Comment:
This is a project that @DamianEdwards runs, so it's up to him to decide what is "better" :smile: